client/patients/get-funded.html

```diff
@@ -23,18 +23,18 @@
 
     <div class="form-group">
       <label for="Password">Password</label>
-      <input class="form-control" id="Password" type="text" ng-model="patient.password" name="password" placeholder="Password" required>
+      <input class="form-control" id="Password" type="password" ng-model="patient.password" name="password" placeholder="Password" required>
     </div>
 
     <div class="form-group">    
       <label for="Condition">Condition</label>
-      <input class="form-control" id="Condition" type="text" ng-model="patient.condition" name="condition" placeholder="Condition" required>
+      <input class="form-control" id="Condition" type="text" ng-model="patient.condition" name="condition" placeholder="Condition" >
     </div>
 
     <div class="form-group">
     
       <label for="Photo">Photo</label>
-      <input class="form-control" id="Photo" type="file" file-model="profilePic" ng-model="patient.photo" name="photo" placeholder="Photo Upload" required>
+      <input onchange="angular.element(this).scope().uploadFile(this)" class="form-control" id="Photo" type="file" file-model="profilePic" ng-model="patient.photo" name="photo" placeholder="Photo Upload">
```

Comment:
@taylorleh @amy-chiu would love to know 1) whats going on here, 2) how'd you figure this out?  super cool, guys!